### PR TITLE
added load more feature to FAQs

### PR DESCRIPTION
**Pull request by AswaniBolisetti**
@Mustafiz04 
I have added load more feature to FAQs, Kindly check it out

https://github.com/user-attachments/assets/dfd7e456-d7ae-4065-bc5f-c785dbf3c178

### CODE REVIEW

**Comment by AswaniBolisetti** [comment]
@Mustafiz04 I have done the work as you said. If user click the 'Know More' button, then all the FAQs will be shown in the seperate page and i have added a button in all faqs page for redirecting to the home page again
Kindly check it out


https://github.com/user-attachments/assets/d72a96e7-97e9-48eb-a0cb-5ce847531b2b

**Comment by AswaniBolisetti** [comment]
@Mustafiz04 
Could you please review and merge it

File: src/data/morefaq.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
+[
+    {
+      "question": "What is GlobeHoppin?",
+      "answer": "GlobeHoppin is a cutting-edge platform designed for travel enthusiasts. It offers a comprehensive suite of tools for discovering unique travel resources, connecting with fellow adventurers, and planning unforgettable journeys across the globe."
+    },
+    {
+      "question": "How can I contact support?",
+      "answer": "Our dedicated support team is always ready to assist you. You can reach us through our Contact page, where you'll find options for email support, live chat, and phone assistance. We aim to respond to all inquiries within 24 hours."
+    },
+    {
```
Comment (Mustafiz04) [inline src/data/morefaq.json:10]: You can merge both faqData and moreFaq into into single fine. On home just display top 5.

File: src/pages/faq.jsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  };
+
+  const goToHome = () => {
+    navigate("/"); // Navigate to the home route
+  };
+
+  return (
+    <section className="bg-gradient-to-br from-gray-900 via-gray-800 to-gray-900 py-12">
+      <div className="mx-auto w-full max-w-7xl px-4"> {/* Changed px-45 to px-4 for Tailwind */}
+        <h2 className="text-5xl font-bold text-white mb-8 text-center">All FAQs</h2>
```
Comment (Mustafiz04) [inline src/pages/faq.jsx:20]: I believe we already have FAQ component you can use that in both places.
One home page use prop to check how many faq to show by default show all.

**Review by Mustafiz04 — CHANGES_REQUESTED** [review]

**Comment by AswaniBolisetti** [comment]
I tried to do it with the same data file and same component, but i couldn't do it,

**Comment by Mustafiz04** [comment]
> I tried to do it with the same data file and same component, but i couldn't do it, so i created a seperate route for faq which uses morefaq.json file to display all the faqs

You can, it require small logic. It will good learning for you.
We don't have to repeat the code.

Also rebase and resolve merge conflict.

**Comment by AswaniBolisetti** [comment]
@Mustafiz04 
Thank you, I deleted the duplicate data file and created faq page using the same json file, now you can check it out, I resolved the conflicts too

**Review by Mustafiz04 — APPROVED** [review]

**Comment by Mustafiz04** [comment]
Resolve the merge conflict again, please.

**Comment by AswaniBolisetti** [comment]
@Mustafiz04 resolved, check once

**Comment by AswaniBolisetti** [comment]
> @Mustafiz04 resolved, check once

@Mustafiz04 could you please review and merge it

**Review by Mustafiz04 — APPROVED** [review]

**Pull request merged**